src/helpers/imageMapping.js

```diff
@@ -0,0 +1,23 @@
+import lmb from '../assets/imgs/lmb.svg'
```

Comment:
There is a mapping file in utils directory. We should strive to keep these in the same directory I will approve but those need to me moved accordingly.